test/solidity/Facets/StandardizedCallFacet.t.sol

```diff
@@ -0,0 +1,64 @@
+// SPDX-License-Identifier: Unlicense
+pragma solidity 0.8.17;
+
+import { Test } from "forge-std/Test.sol";
+import { StandardizedCallFacet } from "lifi/Facets/StandardizedCallFacet.sol";
+import { LiFiDiamond } from "lifi/LiFiDiamond.sol";
+import { DiamondTest } from "../utils/DiamondTest.sol";
+
+interface Diamond {
+    function standardizedCall(bytes calldata _data) external payable;
+
+    function registerPeripheryContract(
+        string calldata _name,
+        address _contract
+    ) external;
+
+    function getPeripheryContract(
+        string calldata _name
+    ) external view returns (address);
+}
+
+contract NotAContract {
+    function notAFunction() external {}
+}
+
+contract StandardizedCallFacetTest is DiamondTest, Test {
+    Diamond internal diamond;
+    StandardizedCallFacet internal standardizedCallFacet;
+
+    function setUp() public {
+        LiFiDiamond tmpDiamond = createDiamond();
+        standardizedCallFacet = new StandardizedCallFacet();
+
+        bytes4[] memory functionSelectors = new bytes4[](1);
+        functionSelectors[0] = standardizedCallFacet.standardizedCall.selector;
+        addFacet(
+            tmpDiamond,
+            address(standardizedCallFacet),
+            functionSelectors
+        );
+
+        diamond = Diamond(address(tmpDiamond));
+    }
+
+    function testCanCallOtherFacet() public {
```

Comment:
I do not really understand this test at all. 

In my understanding, the StandardizedFacet is supposed to forward calls to FACETS, not to periphery. 
This test seems to me like a test of the peripheryRegistryFacet. 

Shouldnt the test setup more be like:

- create diamond
- add FooFacet
- make call to FooFacet.fooFunction() through standardizedFacet with selector of fooFunction
?

Reply:
PeripheryRegistry facet is a facet. All the test cares about is whether we can make a call that uses the same context as any other call to the diamond would.

Reply:
Right, I understand your thinking.

I find it confusing though that we test this facet in a way that it's not intended to being used. 
The intended use is to initiate a bridge tx through this standardized entry point. 
A test case that would mimic the actual usage would make more sense imho, even though the effects may be the same. 
It would also be nice to showcase this for a couple of different facets, not just for one to make the test more robust.